### PR TITLE
add: indexed.xyz project

### PR DESCRIPTION
**Pull request by patsissons**

### CODE REVIEW

**Comment by hazemdraosha111** [comment]
H

**Comment by patsissons** [comment]
Hey @ccerv1, prepped this PR for your review to add our project to the list. let me know if you need to make any changes, if i forgot to add anything, or if you need any clarification about our project (check it out at https://indexed.xyz if you want to see it in action, or the rPGF application with more info [here](https://round3.optimism.io/projects/0x7f79282eea83f7969d81ae6254a31fc755e299c5bce901c2e241f60564159fbe))

File: data/projects/i/indexed-xyz.yaml
[the remark this answers is not in the document]
Comment (patsissons) [reply data/projects/i/indexed-xyz.yaml:1]: definitely, i meant to change that ahead of pinging you 😅

**Review by ccerv1 — APPROVED** [review]
Looks good! TY!

**Pull request merged**